projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/JobImageBuilder.java

```diff
@@ -158,6 +165,12 @@ public boolean buildImage(
       }
     }
 
+    if (REGISTRY_TYPE_ECR.equalsIgnoreCase(registryType)) {
+      ecrRegistryInterface.createRepository(
+          DockerImageName.getImagePath(dockerRepositoryUrl) + "/" + dataJob.getName(),
```

Comment:
I think it is good idea to create a method that builds an image name.

Reply:
What do you mean?